pyproject.toml

```diff
@@ -102,16 +101,6 @@ perf = ["orjson"]
 importers = ["polars", "rich", "filelock", "mlflow", "tenacity"]
 reports = ["pydantic>=2.0.0"]
 
-[tool.setuptools]
-zip-safe = false
-include-package-data = true
-packages = ["wandb"]
-package-dir = { "wandb" = "wandb" }
-package-data = { "wandb" = ["py.typed"] }
-
-[tool.setuptools.dynamic]
-version = { attr = "wandb.__version__" }
```

Comment:
probably need to fix it for hatch, since it is currently dynamic

Reply:
It's specified in `hatch.toml`. Hatch parses the file, like bump2version, rather than loading it and reading an attribute like setuptools.